onnx/reference/reference_evaluator.py

```diff
@@ -11,6 +11,7 @@
 from onnx.defs import onnx_opset_version
 from onnx.onnx_pb import FunctionProto, GraphProto, ModelProto, NodeProto, TypeProto
 from onnx.reference.op_run import OpRun, RuntimeContextError
+from onnx.reference.ops_optimized import Conv as ConvOptimized
```

Comment:
Nit: instead of hard-coding the list of optimized ops here, may be better to import a list of optimized ops from `onnx.reference.ops_optimized` ? So, users need to update only one place when adding a new op there?

Reply:
done